Prog/Lgunit- ACS Table 1B.sas

```diff
@@ -0,0 +1,212 @@
+	      else aff1000median=0;
+	 if  pct3baffordable1000 >=0.75 then  aff1000threequarter=1;
+	      else aff1000threequarter=0;
+     if   pct3baffordable1500 >=0.5 then aff1500median=1;
+	      else aff1500median=0;
+	 if  pct3baffordable1500 >=0.75 then  aff1500threequarter=1;
+	      else aff1500threequarter=0;
+
+run;
+
+
+data ACScharacteristics;
+     set ACS.acs_2012_16_dc_sum_tr_tr10;
+	 keep geo2010 popaloneh_&_years.  popblacknonhispbridge_&_years.  popalonew_&_years.  popwithrace_&_years. pop25andoveryears_&_years.
+          pop25andoverwcollege_&_years. pop25andoverwouths_&_years. popunemployed_&_years. poppoorpersons_&_years. totpop_&_years. 
+	      famincomelt75k_&_years. numfamilies_&_years. nonfamilyhhtot_&_years. medfamincm_&_years. numhshlds_&_years.
+		  pop25andoveryears&_years. popincivlaborforce_&_years.	numrenteroccupiedhu_&_years.	
+          NumRtOHU1u_&_years. NumRtOHU2to4u_&_years. NumRtOHU5to9u_&_years. NumRtOHU10to19u_&_years. NumRtOHU20plusu_&_years.
+          pctnonwht pcthispan pctnonhisblk pctcollege pctwouths pctunemployed pctpoverty pctfambelow75000 pctnonfam
+		  rentersinglefam renter2to4 renter5to9 renter10to19 renter20plus 
+
+;  
+
+pctnonwht= (popwithrace_&_years. -popalonew_&_years.)/popwithrace_&_years.*100;
+pcthispan= (popaloneh_&_years.)/popwithrace_&_years.*100;
+pctnonhisblk= (popblacknonhispbridge_&_years.)/popwithrace_&_years.*100;
+pctcollege= (pop25andoverwcollege_&_years.)/pop25andoveryears_&_years.*100;
+pctwouths= (pop25andoverwouths_&_years.)/pop25andoveryears_&_years.*100;
+pctunemployed= (popunemployed_&_years.)/popincivlaborforce_&_years.*100;
+pctpoverty= (poppoorpersons_&_years.)/totpop_&_years. *100;
+pctfambelow75000 = (famincomelt75k_&_years.)/numfamilies_&_years.*100;
+pctnonfam= (nonfamilyhhtot_&_years.)/numhshlds_&_years.*100;
+rentersinglefam=  (NumRtOHU1u_&_years.)/numrenteroccupiedhu_&_years.*100;
+renter2to4 = (NumRtOHU2to4u_&_years.) /numrenteroccupiedhu_&_years.*100;
+renter5to9= (NumRtOHU5to9u_&_years.)/numrenteroccupiedhu_&_years.*100;
+renter10to19 = (NumRtOHU10to19u_&_years.) /numrenteroccupiedhu_&_years.*100;
+renter20plus = (NumRtOHU20plusu_&_years. )/numrenteroccupiedhu_&_years.*100
+
+;
+
+run;
+
+data crimedata;
+     set Police.Crimes_sum_tr10;
+	 keep geo2010 crimes_pt1_property_2017 crimes_pt1_violent_2017 crime_rate_pop_2017 pctpropertycrime pctviolentcrime ;
+     pctpropertycrime= crimes_pt1_property_2017/crime_rate_pop_2017*100;
+     pctviolentcrime=crimes_pt1_violent_2017/crime_rate_pop_2017*100;
+	 ;
+run;
+
+data prenatal;
+     set vital.Births_2016;
+     keep births_prenat_adeq births_total pctprenatal geo2010;
+	 pctprenatal= births_prenat_adeq/births_total*100
+	 ;
+run;
+
+data ressale;
+     set realprop.sales_res_clean ;
+     keep geo2010 saleprice saledate saleyear ;
+	 saleyear=year(saledate)
+	 ;
+run;
+proc sort data= ressale;
+by geo2010;
+run;
+
+proc means median data = ressale (where=(saleyear=2017)); 
```

Comment:
i don't think median is appearing on the output file.

Reply:
also the output file has several rows per tract and isn't merging correctly later.